lte/gateway/python/magma/subscriberdb/crypto/lte.py

```diff
@@ -50,3 +61,24 @@ def generate_eutran_vector(self, key, opc, sqn, plmn):
             kasme (bytes): 256 bit base network authentication code
         """
         pass
+
+    
+    
+
+    @abc.abstractmethod
+    def generate_m5Gran_vector(self, key: bytes, opc: bytes, sqn: int, snni: bytes)->FiveGRanAuthVector:
+        """
+        Generate the NGRAN key vector.
+        Args:
+            key (bytes): 128 bit subscriber key
+            opc (bytes): 128 bit operator variant algorithm configuration field
+            sqn (int): 48 bit sequence number
+            snni (bytes): 32 bit serving network name consisting of MCC and MNC
+        Returns:
+            rand (bytes): 128 bit random challenge
+            xres (bytes): 128 bit expected result
+            autn (bytes): 128 bit authentication token
+            kseaf (bytes): 256 bit base network authentication code
+            FiveGRanAuthVector (NamedTuple): (rand, xres_star, autn, kseaf)
+        """
+        return FiveGRanAuthVector()
```

Comment:
fill the `FiveGRanAuthVector` tuple here?

Reply:
I've addressed the comment.